packages/terra-form-select/src/combobox/Menu.jsx

```diff
@@ -104,7 +103,9 @@ class Menu extends React.Component {
   constructor(props) {
     super(props);
 
-    this.state = {};
+    this.state = {
+      closedViaKeyEvent: false,
```

Comment:
Can you provide link of code line where we are changing the value of this state from its initial value

Reply:
<img width="870" alt="Screenshot 2023-11-23 at 6 39 32 PM" src="https://github.com/cerner/terra-core/assets/132886427/813808c9-d9b1-40f4-89f2-f468de03b073">

Reply:
https://github.com/cerner/terra-core/blob/35a6252dbcc002b5259c5e9d7f5018db66b4ca7d/packages/terra-form-select/src/combobox/Menu.jsx#L221